app/cmd/plugin_search.go

```diff
@@ -72,13 +72,18 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 func (o *PluginSearchOption) Output(obj interface{}) (data []byte, err error) {
 	if data, err = o.OutputOption.Output(obj); err != nil {
 		pluginList := obj.([]client.AvailablePlugin)
-		table := util.CreateTable(os.Stdout)
-		table.AddRow("number", "name", "installed", "title")
-		for i, plugin := range pluginList {
-			table.AddRow(fmt.Sprintf("%d", i), plugin.Name,
-				fmt.Sprintf("%v", plugin.Installed), plugin.Title)
+		if len(pluginList) == 0 {
+			fmt.Printf("Sorry, you search plugin is unavailable")
```

Comment:
I believe that a new line with this output will be helpful.

Reply:
Yes, when I search plugin but no found plugin  I saw show me "number name installed title", So I think maybe we can modify it to this.